registry/etcdv3/listener_test.go

```diff
@@ -66,6 +69,10 @@ func (suite *RegistryTestSuite) SetupSuite() {
 // stop etcd server
 func (suite *RegistryTestSuite) TearDownSuite() {
 	suite.etcd.Close()
+	// clean the etcd workdir
+	if err := exec.Command("rm", "-rf", defaultEtcdV3WorkDir).Run(); err != nil {
```

Comment:
don't use ```rm``` , no ```rm``` command in windows.........

Reply:
Yes, i fix in latest commit.